Frameworks/ShortcutRecorder.framework/Versions/A/Headers/SRRecorderCell.h

```diff
@@ -12,7 +12,7 @@
 //      Jamie Kirkpatrick
 
 #import <Cocoa/Cocoa.h>
-#import <ShortcutRecorder/SRCommon.h>
+#import "SRCommon.h"
```

Comment:
Is this actually correct with modern framework bridging? I mean, if it works, then ship it, just interesting change to me! :)

Reply:
Yeah I thought that, but, that's what Xcode did. It works.